todo-list/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Need to set alignment of add task button at center
Improve text box design (Match with UI)